Document we don't create PodDisruptionBudgets (#480)
stackabletech/opa-operator: docs/modules/opa/pages/usage-guide/operations/pod-disruptions.adoc

= Allowed Pod disruptions

> For OPA clusters we don't deploy any PodDisruptionBudgets (PDBs), as there is one instance per Kubernetes node running.

Reviewer: ```suggestion
For OPA clusters, the operator does not deploy any PodDisruptionBudgets (PDBs), as there is one instance per Kubernetes node running (Daemonset).
```
